Update azure-stack-storage-connect-se.md

xiaofmao opened the pull request:
Update the step of export certificate, which was changed in latest release build.

azurecla commented:
Hi __@JonCanning__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

xiaofmao committed:
Update azure-stack-storage-connect-se.md

Update the step of export certificate, which was changed in latest release build.
